### DISCUSSION TITLE
Cluster connected but no details

### DISCUSSION THREAD

**Discussion by TaraBrownSmith** [MuhammedKalkan/OpenLens#98] [Q&A]
I'm new to OpenLens, having previously used LENS. I installed with scoop. The install appears to be successful however, once connected to my cluster, there are no details. I'm seeing zero pods, deployments, daemon sets, etc. What configuration options should I review? Thanks in advance for your assistance.

OpenLens: 6.3.0
Extension API: 6.3.0
Electron: 19.1.9
Chrome: 102.0.5005.167
Node: 16.14.2

**Comment by Akshay1181** [comment]
Hi @TaraBrownSmith 

Here is the workaround you can follow atleast on windows machine:

Uninstall the existing openlens , also manually delete directory: \AppData\Local\Programs\OpenLens
Delete this as well : \AppData\Local\open-lens-updater
Re-install openlens [v6.2.5](https://github.com/MuhammedKalkan/OpenLens/releases/tag/v6.2.5)
It seems cache is causing this issue, while first time installing, sometime cluster is not getting registered properly.

It worked for me😀,
